baseline.py

```diff
@@ -86,8 +90,9 @@ def evaluate_baseline(self, sim, out_prefix, n_min, n_max, n_inc=10):
 
 """
 
-class baseline(object):
 
+# TODO: Added by JP for Liniting errors, cannot have multiple baseline object definitions
+class baseline_alternate(object):
```

Comment:
@cvstewart Please review this.  There were two definitions of the `baseline` object in `baseline.py`.  I've converted the second one to the name `baseline_alternate`, but I wasn't sure of this was correct.